lib/app/features/about/view/about_view.dart

```diff
@@ -30,28 +32,34 @@ https://gordonferguson.org/wp-content/uploads/2016/11/Final-Main-Header.jpg''';
     return ErrorSnackbarView<void>(
       provider: launchProvider(url: _url.value),
       child: Scaffold(
-        appBar: AppBar(title: const Text('About'), actions: [SettingsIconButton()]),
+        appBar: AppBar(
+            title: const Text('About'),
+            centerTitle: Platform.isAndroid,
+            actions: [SettingsIconButton()]),
         body: AdjustableTextWidget(
           child: MediaQuery.withClampedTextScaling(
             maxScaleFactor: 1.7,
-            child: Center(
-              child: Column(
-                children: [
-                  const SizedBox(height: 30),
-                  WpaImage(_imageUrl),
-                  Html(
-                    data: _resourcesHtml,
-                    shrinkWrap: true,
-                    style: {
-                      '*': Style(
-                        fontSize: FontSize.large,
-                        textAlign: TextAlign.center,
-                      )
-                    },
-                    onLinkTap: (String? url, _, __) =>
-                        (url == _url.value) ? ref.invalidate(launchProvider) : _url.value = url,
-                  ),
-                ],
+            child: SingleChildScrollView(
+              child: Center(
```

Comment:
you might try to see if you an save the center by using mainaxisalignment center or similar?